.gitignore

```diff
@@ -131,3 +131,8 @@ dmypy.json
 .idea
 app/static
 govuk_components
+
+# bash_profile & .DS_Store
```

Comment:
comment

Reply:
Hello world FS-387 would be fine?

Reply:
Hello world security?

Reply:
Commented: # Hello world security FS-387 files